turbopy/diagnostics.py

```diff
@@ -20,8 +20,9 @@ class CSVDiagnosticOutput:
     ----------
     filename : str
        File name for CSV data file.
-    diagnostic_size : int
-       Size of data set to be written to CSV file.
+    diagnostic_size : (int, int)
+       Size of data set to be written to CSV file. First value is the number of
+       time points. Second value is number of spacial points.
```

Comment:
```suggestion
       time points. Second value is number of spatial points.
```

Reply:
While both spellings are acceptable, _spatial_ is the more common spelling.

Reply:
fixed the spelling error in the latest update.